config/config.go

```diff
@@ -35,9 +47,32 @@ func getUserConfigDir() (string, error) {
 	}
 }
 
-func ParseConfigFile() Config {
+func parseConfigFromFile(configPath string, config *Config) error {
+	data, err := os.ReadFile(configPath)
+	if err != nil {
+		return nil
+	}
+	err = toml.Unmarshal(data, config)
+	if err != nil {
+		return fmt.Errorf("Error parsing config file: %s", err)
+	}
+
+	if len(config.IncludedPaths) > 0 {
+		for _, includePath := range config.IncludedPaths {
+			includeConfig := Config{}
+			if err := parseConfigFromFile(includePath.Path, &includeConfig); err != nil {
```

Comment:
Please add a benchmark test and figure if there's a significant drop in performance if someone uses this feature a lot. I'd like to warn people in the README if heavy use of this feature causes performance problems.

Reply:
It makes sense. I will write some benchmark tests.

Reply:
```
go test -bench=. ./config/config_test.go -benchtime=10s -benchmem
goos: darwin
goarch: arm64
BenchmarkParseConfigFile/ParseConfigFile_1-8              514300             23670 ns/op            5176 B/op         52 allocs/op
BenchmarkParseConfigFile/ParseConfigFile_10-8              98485            122762 ns/op           27536 B/op        256 allocs/op
BenchmarkParseConfigFile/ParseConfigFile_100-8             10000           1171631 ns/op          249921 B/op       2242 allocs/op
BenchmarkParseConfigFile/ParseConfigFile_1000-8              910          12548132 ns/op         2515669 B/op      22050 allocs/op
BenchmarkParseConfigFile/ParseConfigFile_10000-8              73         164291629 ns/op        25620936 B/op     220066 allocs/op
PASS
ok      command-line-arguments  83.454s
```

Reply:
it affects the performance if there are a thousand separate extended config files. But I think the user only needs one extended_config in most cases. So I don't think there will be any performance issues.